
Epoch 1/30

65/66 [============================>.] - ETA: 0s - loss: 0.6906 - mean_absolute_error: 0.1325
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
66/66 [==============================] - 9s 78ms/step - loss: 0.6906 - mean_absolute_error: 0.1325 - val_loss: 0.6887 - val_mean_absolute_error: 0.1201
Epoch 2/30
55/66 [========================>.....] - ETA: 0s - loss: 0.6868 - mean_absolute_error: 0.1272
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

65/66 [============================>.] - ETA: 0s - loss: 0.6862 - mean_absolute_error: 0.1264
66/66 [==============================] - 4s 64ms/step - loss: 0.6862 - mean_absolute_error: 0.1263 - val_loss: 0.6824 - val_mean_absolute_error: 0.1126
Epoch 3/30
51/66 [======================>.......] - ETA: 0s - loss: 0.6724 - mean_absolute_error: 0.1109
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

65/66 [============================>.] - ETA: 0s - loss: 0.6698 - mean_absolute_error: 0.1045
66/66 [==============================] - 4s 63ms/step - loss: 0.6698 - mean_absolute_error: 0.1045 - val_loss: 0.6566 - val_mean_absolute_error: 0.0766
Epoch 4/30
45/66 [===================>..........] - ETA: 0s - loss: 0.6447 - mean_absolute_error: 0.0719
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

65/66 [============================>.] - ETA: 0s - loss: 0.6433 - mean_absolute_error: 0.0674
66/66 [==============================] - 4s 68ms/step - loss: 0.6433 - mean_absolute_error: 0.0674 - val_loss: 0.6410 - val_mean_absolute_error: 0.0522
Epoch 5/30
35/66 [==============>...............] - ETA: 1s - loss: 0.6330 - mean_absolute_error: 0.0510
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

65/66 [============================>.] - ETA: 0s - loss: 0.6322 - mean_absolute_error: 0.0483
66/66 [==============================] - 4s 64ms/step - loss: 0.6322 - mean_absolute_error: 0.0483 - val_loss: 0.6361 - val_mean_absolute_error: 0.0416
Epoch 6/30
31/66 [=============>................] - ETA: 1s - loss: 0.6277 - mean_absolute_error: 0.0424
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

65/66 [============================>.] - ETA: 0s - loss: 0.6289 - mean_absolute_error: 0.0413
66/66 [==============================] - 4s 65ms/step - loss: 0.6289 - mean_absolute_error: 0.0413 - val_loss: 0.6347 - val_mean_absolute_error: 0.0390
Epoch 7/30

65/66 [============================>.] - ETA: 0s - loss: 0.6278 - mean_absolute_error: 0.0389
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
66/66 [==============================] - 4s 65ms/step - loss: 0.6278 - mean_absolute_error: 0.0389 - val_loss: 0.6340 - val_mean_absolute_error: 0.0369
Epoch 8/30

65/66 [============================>.] - ETA: 0s - loss: 0.6272 - mean_absolute_error: 0.0375
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
66/66 [==============================] - 4s 63ms/step - loss: 0.6272 - mean_absolute_error: 0.0375 - val_loss: 0.6336 - val_mean_absolute_error: 0.0359
Epoch 9/30

65/66 [============================>.] - ETA: 0s - loss: 0.6268 - mean_absolute_error: 0.0366
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
66/66 [==============================] - 4s 65ms/step - loss: 0.6268 - mean_absolute_error: 0.0366 - val_loss: 0.6334 - val_mean_absolute_error: 0.0357
Epoch 10/30

59/66 [=========================>....] - ETA: 0s - loss: 0.6265 - mean_absolute_error: 0.0359
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(

66/66 [==============================] - 4s 63ms/step - loss: 0.6265 - mean_absolute_error: 0.0359 - val_loss: 0.6330 - val_mean_absolute_error: 0.0341
Epoch 11/30
55/66 [========================>.....] - ETA: 0s - loss: 0.6266 - mean_absolute_error: 0.0352
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

65/66 [============================>.] - ETA: 0s - loss: 0.6262 - mean_absolute_error: 0.0350
66/66 [==============================] - 4s 63ms/step - loss: 0.6262 - mean_absolute_error: 0.0350 - val_loss: 0.6328 - val_mean_absolute_error: 0.0334
Epoch 12/30
49/66 [=====================>........] - ETA: 0s - loss: 0.6263 - mean_absolute_error: 0.0345
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

65/66 [============================>.] - ETA: 0s - loss: 0.6259 - mean_absolute_error: 0.0342
66/66 [==============================] - 4s 66ms/step - loss: 0.6259 - mean_absolute_error: 0.0342 - val_loss: 0.6326 - val_mean_absolute_error: 0.0330
Epoch 13/30
45/66 [===================>..........] - ETA: 0s - loss: 0.6238 - mean_absolute_error: 0.0337
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

65/66 [============================>.] - ETA: 0s - loss: 0.6257 - mean_absolute_error: 0.0335
66/66 [==============================] - 4s 63ms/step - loss: 0.6257 - mean_absolute_error: 0.0335 - val_loss: 0.6324 - val_mean_absolute_error: 0.0326
Epoch 14/30
41/66 [=================>............] - ETA: 0s - loss: 0.6244 - mean_absolute_error: 0.0330
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

65/66 [============================>.] - ETA: 0s - loss: 0.6255 - mean_absolute_error: 0.0328
66/66 [==============================] - 4s 65ms/step - loss: 0.6255 - mean_absolute_error: 0.0328 - val_loss: 0.6321 - val_mean_absolute_error: 0.0313
Epoch 15/30
33/66 [==============>...............] - ETA: 1s - loss: 0.6252 - mean_absolute_error: 0.0320
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

65/66 [============================>.] - ETA: 0s - loss: 0.6254 - mean_absolute_error: 0.0319
66/66 [==============================] - 4s 65ms/step - loss: 0.6253 - mean_absolute_error: 0.0319 - val_loss: 0.6319 - val_mean_absolute_error: 0.0305
Epoch 16/30
27/66 [===========>..................] - ETA: 1s - loss: 0.6222 - mean_absolute_error: 0.0315
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

65/66 [============================>.] - ETA: 0s - loss: 0.6251 - mean_absolute_error: 0.0310
66/66 [==============================] - 4s 63ms/step - loss: 0.6250 - mean_absolute_error: 0.0310 - val_loss: 0.6317 - val_mean_absolute_error: 0.0299
Epoch 17/30

65/66 [============================>.] - ETA: 0s - loss: 0.6248 - mean_absolute_error: 0.0302
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
66/66 [==============================] - 4s 66ms/step - loss: 0.6248 - mean_absolute_error: 0.0302 - val_loss: 0.6315 - val_mean_absolute_error: 0.0289
Epoch 18/30

65/66 [============================>.] - ETA: 0s - loss: 0.6245 - mean_absolute_error: 0.0292
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
66/66 [==============================] - 4s 63ms/step - loss: 0.6245 - mean_absolute_error: 0.0292 - val_loss: 0.6314 - val_mean_absolute_error: 0.0294
Epoch 19/30

63/66 [===========================>..] - ETA: 0s - loss: 0.6241 - mean_absolute_error: 0.0281
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
66/66 [==============================] - 4s 63ms/step - loss: 0.6242 - mean_absolute_error: 0.0280 - val_loss: 0.6309 - val_mean_absolute_error: 0.0266
Epoch 20/30

55/66 [========================>.....] - ETA: 0s - loss: 0.6235 - mean_absolute_error: 0.0265
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

65/66 [============================>.] - ETA: 0s - loss: 0.6239 - mean_absolute_error: 0.0265
66/66 [==============================] - 4s 66ms/step - loss: 0.6239 - mean_absolute_error: 0.0265 - val_loss: 0.6307 - val_mean_absolute_error: 0.0268
Epoch 21/30
51/66 [======================>.......] - ETA: 0s - loss: 0.6228 - mean_absolute_error: 0.0254
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

65/66 [============================>.] - ETA: 0s - loss: 0.6236 - mean_absolute_error: 0.0252
66/66 [==============================] - 4s 64ms/step - loss: 0.6236 - mean_absolute_error: 0.0252 - val_loss: 0.6304 - val_mean_absolute_error: 0.0246
Epoch 22/30
45/66 [===================>..........] - ETA: 0s - loss: 0.6231 - mean_absolute_error: 0.0241
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

65/66 [============================>.] - ETA: 0s - loss: 0.6234 - mean_absolute_error: 0.0242
66/66 [==============================] - 4s 66ms/step - loss: 0.6234 - mean_absolute_error: 0.0242 - val_loss: 0.6302 - val_mean_absolute_error: 0.0237
Epoch 23/30
35/66 [==============>...............] - ETA: 1s - loss: 0.6239 - mean_absolute_error: 0.0237
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

65/66 [============================>.] - ETA: 0s - loss: 0.6232 - mean_absolute_error: 0.0235
66/66 [==============================] - 4s 64ms/step - loss: 0.6232 - mean_absolute_error: 0.0235 - val_loss: 0.6301 - val_mean_absolute_error: 0.0231
Epoch 24/30
31/66 [=============>................] - ETA: 1s - loss: 0.6241 - mean_absolute_error: 0.0232
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

65/66 [============================>.] - ETA: 0s - loss: 0.6231 - mean_absolute_error: 0.0231
66/66 [==============================] - 4s 67ms/step - loss: 0.6231 - mean_absolute_error: 0.0231 - val_loss: 0.6300 - val_mean_absolute_error: 0.0224
Epoch 25/30

65/66 [============================>.] - ETA: 0s - loss: 0.6230 - mean_absolute_error: 0.0227
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
66/66 [==============================] - 4s 67ms/step - loss: 0.6230 - mean_absolute_error: 0.0227 - val_loss: 0.6299 - val_mean_absolute_error: 0.0220
Epoch 26/30

61/66 [==========================>...] - ETA: 0s - loss: 0.6232 - mean_absolute_error: 0.0221
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

65/66 [============================>.] - ETA: 0s - loss: 0.6230 - mean_absolute_error: 0.0221
66/66 [==============================] - 4s 65ms/step - loss: 0.6229 - mean_absolute_error: 0.0221 - val_loss: 0.6298 - val_mean_absolute_error: 0.0220
Epoch 27/30
51/66 [======================>.......] - ETA: 0s - loss: 0.6230 - mean_absolute_error: 0.0219
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

65/66 [============================>.] - ETA: 0s - loss: 0.6228 - mean_absolute_error: 0.0218
66/66 [==============================] - 5s 69ms/step - loss: 0.6228 - mean_absolute_error: 0.0218 - val_loss: 0.6297 - val_mean_absolute_error: 0.0214
Epoch 28/30
39/66 [================>.............] - ETA: 1s - loss: 0.6244 - mean_absolute_error: 0.0214
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

66/66 [==============================] - ETA: 0s - loss: 0.6227 - mean_absolute_error: 0.0214
66/66 [==============================] - 4s 67ms/step - loss: 0.6227 - mean_absolute_error: 0.0214 - val_loss: 0.6297 - val_mean_absolute_error: 0.0218
Epoch 29/30
31/66 [=============>................] - ETA: 1s - loss: 0.6233 - mean_absolute_error: 0.0217
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

64/66 [============================>.] - ETA: 0s - loss: 0.6226 - mean_absolute_error: 0.0212
66/66 [==============================] - 4s 66ms/step - loss: 0.6227 - mean_absolute_error: 0.0212 - val_loss: 0.6296 - val_mean_absolute_error: 0.0209
Epoch 30/30

65/66 [============================>.] - ETA: 0s - loss: 0.6226 - mean_absolute_error: 0.0210
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(

66/66 [==============================] - 4s 64ms/step - loss: 0.6226 - mean_absolute_error: 0.0210 - val_loss: 0.6295 - val_mean_absolute_error: 0.0205